src/views/docs/en/guides/extend/plugins.md

```diff
@@ -51,28 +52,26 @@ Plugin authors should create a module that exports an object with properties of
   */
 
 module.exports = {
-  package: async function extendCloudFormation ({ arc, cloudformation, stage='staging', inventory }) {
-  },
-  pluginFunctions: function ({ arc, inventory }) {
-  },
+  package: async function extendCloudFormation ({ arc, cloudformation, stage='staging', inventory, createFunction }) {},
```

Comment:
Maybe just me, but when I see the named function here (`extendCloudFormation`) and the rest are not, this makes me think that name is somehow mandatory.